mapswipe_workers/mapswipe_workers/firebase_to_postgres/transfer_results.py

```diff
@@ -221,12 +240,20 @@ def results_to_file(results, projectId):
                 )
                 continue
 
+            user_group_ids = [
+                user_group_id
+                for user_group_id, is_selected in result.get("userGroups", {}).items()
+                if is_selected
+            ]
             start_time = dateutil.parser.parse(start_time)
             end_time = dateutil.parser.parse(end_time)
             timestamp = end_time
+            results_added = False
 
-            if type(results) is dict:
-                for taskId, result in results.items():
+            if type(result_results) is dict:
+                if result_results:
+                    results_added = True
+                for taskId, result in result_results.items():
```

Comment:
You're reusing `result` as a variable name here, this will cause problems.

Reply:
Good catch.